brigade.js

```diff
@@ -66,14 +69,36 @@ function test() {
   return job;
 }
 
+function testViaDocker() {
+  // Create a new job to run Go tests via Docker
+  // to ensure Docker-based builds are functional
+  var job = new Job("tests-via-docker", "docker:stable-dind");
+  // Set privileged true to enable Docker-in-Docker
+  job.privileged = true;
+  // Set a few environment variables.
+  job.env = {
+    "GO111MODULE": "on",
```

Comment:
Is this env var necessary? 🤔  I always set it in the Makefile so that it is set consistently everywhere. But I also see that it's being set in the Dockerfile right now.

Reply:
Mmm, appears not.  Removed.